src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(toFileName);
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can not create file", e);
+        }
+        writeToFile(countStatistic(getFromFile(fromFileName)),file.getName());
+    }
+
+    private String getFromFile(String fromFilename) {
+        try {
+            StringBuilder sb;
```

Comment:
Don't use abbreviations 
```suggestion
            StringBuilder result;
```